etc/kubernetes/helm/templates/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: enterprise-gateway
+  namespace: enterprise-gateway
+  labels:
+    gateway-selector: enterprise-gateway
+    app: enterprise-gateway
+    component: enterprise-gateway
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      gateway-selector: enterprise-gateway
+  template:
+    metadata:
+      labels:
+        gateway-selector: enterprise-gateway
+        app: enterprise-gateway
+        component: enterprise-gateway
+    spec:
+      # Created by this chart.
+      serviceAccountName: enterprise-gateway-sa
+      containers:
+        - name: enterprise-gateway
+          image: {{ .Values.image }}
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          args: ["--gateway"]
+          env:
+          - name: EG_NAMESPACE
+            value: {{ .Release.Namespace }}
+          - name: EG_KERNEL_CLUSTER_ROLE
+            value: {{ .Values.kernel_cluster_role }}
+          - name: EG_SHARED_NAMESPACE
+            value: {{ if .Values.shared_namespace }}"True"{{ else }}"False"{{ end }}
+          - name: EG_MIRROR_WORKING_DIRS
+            value: {{ if .Values.mirror_working_dirs }}"True"{{ else }}"False"{{ end }}
+          - name: EG_CULL_IDLE_TIMEOUT
+            value: !!str {{ .Values.cull_idle_timeout }}
+          - name: EG_LOG_LEVEL
+            value: {{ .Values.log_level }}
+          - name: EG_KERNEL_LAUNCH_TIMEOUT
+            value: !!str {{ .Values.kernel_launch_timeout }}
+          - name: EG_KERNEL_WHITELIST
+            value: {{ toJson .Values.kernel_whitelist | squote }}
+          ports:
+          - containerPort: 8888
+{{ if .Values.nfs.enabled }}
+          volumeMounts:
+          - name: kernelspecs
+            mountPath: "/usr/local/share/jupyter/kernels"
+        volumes:
+        - name: kernelspecs
+          nfs:
+            server: {{ .Values.internal_server_ip_address }}
```

Comment:
Does this need to be `{{ .Values.nfs.internal_server_ip_address }}`?

Reply:
Yes, good catch! Fixed.